dev/AppLifecycle/AppInstance.cpp

```diff
@@ -337,14 +337,20 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
             // protocol, except the catch-all LaunchActivatedEventArgs case.
             if (!contractArgument.empty())
             {
-                if (contractData.empty())
+                // If the contractData is empty, handle any aliased encoded launches.
```

Comment:
This comment is no longer correct given your change.

Edit: I think if you just reword it to not be about contractData being empty it will make sense.  Maybe just calling out that we're looking for an aliased launches.

Reply:
It is only for unpackaged apps. For packaged apps, it should be empty.